### ISSUE TITLE
Don't split up Jobs when not clustering

### ISSUE THREAD

**Issue by thombergs** [reflectoring/coderadar#158]
Currently, there are multiple `ScanFilesJob`s and `AnalyzeCommitJob`s queued at once, so that multiple server nodes may work them in parallel. However, if there is only one server node, you actually lose time, since the job engine only looks for new jobs every 100ms by default. So, in the worst case, we lose 100ms per job.

1. introduce a new boolean config param `coderadar.jobs.clustered`
2. if the new config param is set to true, queue jobs as before
3. if the new config param is set to false (default), instead of a `ScanFilesJob` queue a new `ScanAllFilesJob` and instead of `AnalyzeCommitJob` queue a new `AnalyzeAllCommitsJob`
4. implement execution for these jobs to do all work at once instead of split up between jobs

**Comment by rzpio** [comment]
The job framework is dropped (see #204), so this is no longer needed. If a job framework is added again, the issue can be opened again.
